modules/standard/Random.chpl

```diff
@@ -549,30 +809,19 @@ module Random {
      and the paper, "PCG: A Family of Simple Fast Space-Efficient Statistically
      Good Algorithms for Random Number Generation" by M.E. O'Neill.
 
-     Besides a Chapel :class:`RandomStream` interface, this module includes a
-     number of low-level PCG random functions:
-
-       * :record:`pcg_setseq_64_xsh_rr_32_rng` (the default PCG RNG)
-       * :record:`pcg_setseq_64_rxs_m_xs_64_rng`
-       * :record:`pcg_setseq_32_rxs_m_xs_32_rng`
-       * :record:`pcg_setseq_16_rxs_m_xs_16_rng`
-       * :record:`pcg_setseq_8_rxs_m_xs_8_rng`
-       * :record:`pcg_setseq_N_rxs_m_xs_N_rng` which is a generalization of the
-         above
-
-     These names come from the PCG paper and reference implementations.  The
-     first integer is the number of state bits, and the last integer is the
-     number of output bits. The other parts describe the permutation function in
-     the LCG.
-
      It also includes some Chapel-specific features, such as generating real,
      imag, and complex numbers; and generating numbers in a range in parallel.
      These features are not available in the reference implementations of PCG.
 
+     .. note::
+
+       This interface provided by this module is expected to change.
+
   */
   module PCGRandom {
 
     use RandomSupport;
+    use PCGRandomLib;
```

Comment:
This is the type of use that I think we'd want to be a 'private use' (non-transitive) if/when we get that concept.  In reviewing code, I usually encourage people to push use's like this into the scopes where the PCGRandomLib symbols are used directly (or to use the fully-qualified names to refer to such symbols) in order to avoid bleeding symbols out of the module unintentionally, but given how widespread this probably is in this routine and the timeframe, maybe that's not reasonable.